<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAuthWidget</class>
 <widget class="QWidget" name="UserAuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#UserAuthWidget {
            background-image: url(&quot;:/pictures/back.png&quot;);
}</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>371</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="scrollArea_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);
border-radius: 10px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Участники тестирования</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QPushButton" name="auth_pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>431</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}</string>
   </property>
   <property name="text">
    <string>Начать тестирование с выбранным участником</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}</string>
   </property>
   <property name="text">
    <string>Добавить участника</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}</string>
   </property>
   <property name="text">
    <string>Удалить участника</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_pushButton">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>360</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}</string>
   </property>
   <property name="text">
    <string>Редактировать данные</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
